assets/app.js

```diff
@@ -0,0 +1,33 @@
+/* global hoodie */
+
+document.body.addEventListener('submit', function (event) {
```

Comment:
Second thought it'd be nicer to add an id to the form and do something like
```js
document.getElementById('tracker').addEventListener('submit', function () {})
```

I'll add comments If I get the chance.